### ISSUE TITLE
Persformer not trainable in orbit5k

### ISSUE THREAD

**Issue by sehunfromdaegu** [giotto-ai/giotto-deep#137]
**Describe the bug**

Thanks for sharing this wonderful code.
One weird thing I've noticed is that the Persformer in /examples/orbit_5k_pipeline.ipynb is not trainable.
I have changed the Persformer hyperparameters and increased the dataset size, but the training accuracy remains around 20%, which is similar to random guessing.

I suspect there might be an issue with the Persformer code.

**Comment by sehunfromdaegu** [comment]
Maybe "Residual connections" described in Section 4.1 is missing in the code.
The paper reports that without the "Residual connections", the model was not trainable.
